src/shared/form_fields/member.js

```diff
@@ -149,12 +142,13 @@ const validate = (values) => {
   const add_test = (tests, key) =>
     assoc(key, options[key] ? selected : exists, tests)
 
-  const req = id ? required :
-    new_required.concat(
+  const req = (id ? required : new_required)
+    .concat(
```

Comment:
The way this is done at the moment only the changes done on the member's view are sync'd up. This is because I applied the changes on the Members Controller, which is what the action on the member side points to. I could take this functionality to the beforeUpdate function in the model though. Do you think that is a good idea?